src/Badge/Model/UseCase/CreateRequireBadge.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/*
+ * This file is part of the badge-poser package.
+ *
+ * (c) PUGX <http://pugx.github.io/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace App\Badge\Model\UseCase;
+
+use App\Badge\Model\CacheableBadge;
+use App\Badge\Model\Package;
+use App\Badge\Model\PackageRepositoryInterface;
+
+/**
+ * Create the 'require' image with the standard Font and standard Image.
+ */
+final class CreateRequireBadge extends BaseCreatePackagistImage
+{
+    public const COLOR = '787CB5';
+    public const TEXT_NO_REQUIRE = ' - ';
+
+    private const TTL_DEFAULT_MAXAGE = CacheableBadge::TTL_ONE_HOUR;
```

Comment:
maybe we can even store the badge for longer, since it shouldn't change that often

Reply:
could change how could the version name change... the require are taken from the latest stable version.